src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -0,0 +1,85 @@
+package com.rtb.projectmanagementtool.auth;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.users.*;
+import com.rtb.projectmanagementtool.user.*;
+import java.io.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+
+public class AuthOps {
+
+  // private DatastoreService datastore;
+  private UserController controller;
+  public String cookieName;
+  public String cookieValue;
+  public Cookie currCookie;
+
+  public AuthOps(UserController controller) {
```

Comment:
As per discussion, this constructor is for testing only so we can pass a mock UserController in.  Please add a comment stating this.  Also please add a default constructor (with no params) that builds a non-test controller